plugin/storage/es/spanstore/writer_test.go

```diff
@@ -235,75 +197,6 @@ func TestSpanIndexName(t *testing.T) {
 	assert.Equal(t, "jaeger-service-1995-04-21", serviceIndexName)
 }
 
-func TestCheckAndCreateIndex(t *testing.T) {
```

Comment:
what's the argument for deleting this test?

Reply:
It does not affect test coverage, I think some of them were repeating stuff